src/internet_identity/src/storage/tests.rs

```diff
@@ -29,15 +29,27 @@ fn should_report_max_number_of_entries_for_32gb() {
 }
 
 #[test]
-fn should_serialize_header_v7() {
+fn should_deserialize_header_v7() {
```

Comment:
nit: the same test for v7 is named `should_recover_header_from_memory_v7` below, maybe use a consistent pattern for both tests?

Reply:
I realized, the test is duplicate anyway. So I added `should_recover_header_from_memory_v8` but removed `should_deserialize_header_v7` as it is a duplicate of `should_recover_header_from_memory_v7`.